collector/__init__.py

```diff
@@ -130,47 +131,47 @@ def load(self, path: str) -> NoReturn:
         # Retrieve the site details
         try:
             site = Site(site_name)
-            location = GEOGRAPHICAL_LOCATIONS[site]
+            self.locations[site] = GEOGRAPHICAL_LOCATIONS[site]
```

Comment:
What will happen, though, when we allow for multisite? Since the Selector uses these values in the Collector, I am going to make a note and leave this one as is for now, I think... that way we can still have the `try`.